udata_croquemort/checker.py

```diff
@@ -24,13 +24,21 @@ def _format_response(self, response):
             check_date = response.get('updated')
             if check_date:
                 check_date = dateutil.parser.parse(response.get('updated'))
-            return {
+
+            result = {
                 'check:url': response.get('checked-url'),
                 'check:status': status,
                 'check:available': status and status >= 200 and status < 400,
                 'check:date': check_date,
             }
 
+            for header in [
+                'content-type', 'content-length', 'content-md5', 'charset'
```

Comment:
We could pull more headers https://github.com/etalab/croquemort/blob/eb972ebee6a17c16ac838edffdaa1d3c3ea23b6e/croquemort/storages.py#L12-L16